CMakeLists.txt

```diff
@@ -72,6 +72,10 @@ option(ENABLE_VIRTUAL_OPERATIONS "This option usually works with REGENERATE_CLIE
 set(BUILD_ONLY "" CACHE STRING "A semi-colon delimited list of the projects to build")
 set(CPP_STANDARD "11" CACHE STRING "Flag to upgrade the C++ standard used. The default is 11. The minimum is 11.")
 
+set(BUILD_ONLY "lambda" "sns")
```

Comment:
@prateek2211 @yuvalif could you split this change into two?

1. one for removing the stuff not used by Ceph
1. the other for changing the CMakeLists.txt?